rel-eng/tito.props

```diff
@@ -133,6 +133,7 @@ whitelist = foreman
   rubygem-gridster-rails
   rubygem-hammer_cli
   rubygem-hammer_cli_foreman
+  rubygem-foreman_maintain
```

Comment:
@ehelms Btw, this comment has been addressed. Not sure why GitHub hasn't resolved it.